common/app/common/controllers/SunriseController.java

```diff
@@ -18,17 +20,30 @@
 public abstract class SunriseController extends ShopController {
     private final CategoryTree categoryTree;
     private final CountryOperations countryOperations;
+    private final TemplateService templateService;
+    private final CmsService cmsService;
 
-    protected SunriseController(final PlayJavaSphereClient client, final CategoryTree categoryTree, final Configuration configuration) {
+    protected SunriseController(final PlayJavaSphereClient client, final CategoryTree categoryTree,
+                                final Configuration configuration, final TemplateService templateService, final CmsService cmsService) {
         super(client);
         this.categoryTree = categoryTree;
         this.countryOperations = CountryOperations.of(configuration);
+        this.templateService = templateService;
+        this.cmsService = cmsService;
     }
 
     protected final CategoryTree categories() {
         return categoryTree;
     }
 
+    public TemplateService viewService() {
```

Comment:
Yes, this was changed already this morning: https://github.com/sphereio/sphere-sunrise/commit/c7b728ea8795c3407c94e15e9876206b752d1606
The refactoring didn't take into account this method.